src/services/notificationService/registerNotifications.ts

```diff
@@ -0,0 +1,34 @@
+import PushNotificationIOS from '@react-native-community/push-notification-ios';
+import PushNotification from 'react-native-push-notification';
+
+export const registerNotifications = () => {
+  // Registering local notifications
+  PushNotification.configure({
+    onRegister(token) {
+      console.log('TOKEN:', token);
+    },
+
+    onNotification(notification) {
+      console.log('NOTIFICATION:', notification);
+      notification.finish(PushNotificationIOS.FetchResult.NoData);
+    },
+
+    onAction(notification) {
+      console.log('ACTION:', notification.action);
+      console.log('NOTIFICATION:', notification);
+    },
+
+    onRegistrationError(err) {
+      if (err.code !== 3010) {
```

Comment:
What is 3010?

Reply:
That's the error when it's trying to register the app on a simulator. I'll add a comment for clarification.